app/controllers/hosts_controller.rb

```diff
@@ -625,14 +625,8 @@ def taxonomy_scope
     @organization ||= Organization.find_by_id(params[:organization_id]) if params[:organization_id]
     @location     ||= Location.find_by_id(params[:location_id])         if params[:location_id]
 
-    if SETTINGS[:organizations_enabled]
-      @organization ||= Organization.current
-      @organization ||= Organization.my_organizations.first
```

Comment:
They would default to `nil`. 
I think anyhow the correct way to handle the "Any" context is not by defaulting to the first one.